wordfreq/__init__.py

```diff
@@ -257,6 +264,9 @@ def word_frequency(word, lang, wordlist='combined', minimum=0.):
     If a word decomposes into multiple tokens, we'll return a smoothed estimate
     of the word frequency that is no greater than the frequency of any of its
     individual tokens.
+
+    It should be noted that the current tokenizer does not support
+    multi-character Chinese terms.
```

Comment:
This is kind of misleading, I think.  Multi-*character* terms are fine:

```python
In [15]: word_frequency('中文', 'zh')
Out[15]: 6.760829753919819e-05
```

It's multi-word phrases that will not be tokenized:

```python
In [16]: word_frequency('我不会讲中文', 'zh')
Out[16]: 0.0
```

Additionally it may be useful to mention that if you have a Chinese tokenizer you can induce wordfreq to do the right thing simply by putting spaces in yourself:
```python
In [17]: word_frequency('我 不会 讲 中文', 'zh')
Out[17]: 4.3847950843537636e-05
```